### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by iglesias** [dso-org/deep-symbolic-optimization#91]
Hi,

the summary of the error is
```
  Installing build dependencies ... done                                                                                                          
  Getting requirements to build wheel ... error                                                                                                   
  error: subprocess-exited-with-error                 
                                                                                                                                                  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1                                                         
  ╰─> [59 lines of output]                                                                                                                        
      <string>:5: _DeprecatedInstaller: setuptools.installer and fetch_build_eggs are deprecated.
      !!                                                                 
                                                                                                                                                  
              ********************************************************************************
              Requirements should be satisfied by a PEP 517 installer.
              If you are using pip, you can try `pip install --use-pep517`.                                                    
              ********************************************************************************
```
reproduced with Python 3.11.

**Comment by iglesias** [comment]
A bit of an update before closing the issue, I am porting it to torch using Python 3.11.8, trying to remove the dependency on tensorflow 1.14. The error above is not an using starting from the scratch the port.

The code is turning very helpful to understand the methods, it is great to see this kind of research 👏

**Comment by brendenpetersen** [comment]
@iglesias Any chance you want to share your pytorch implementation? :)

**Comment by iglesias** [comment]
Are you interested in an incomplete port?

Yesterday I quickly estimated that it is only 1% ported (around the state manager). Unfortunately, I can't prioritize the port to devote the time it needs and finish it within some months. My motivation is just recreational to see if state-of-the-art (Deep) Symbolic Regression can solve one of the 2023 Advent of Code problems by reducing it to a regression, instead of working out the equation with pen and paper.

Since I couldn't run the code initially, I started the port and spent a few hours to get an idea of what it would require. So, in case I would be the only developer, I am afraid the port won't be finished. Now, I have also managed to run it using tensorflow 1.14 with the Dockerfile [in this other issue](https://github.com/dso-org/deep-symbolic-optimization/issues/84#issuecomment-1858844858).
